pkg/storage/bloom/bloom-shipper/bloom_shipper.go

```diff
@@ -0,0 +1,304 @@
+package bloom_shipper
```

Comment:
Files from the `bloom_shipper` package have been copied from https://github.com/grafana/loki/pull/10697 and need to be replaced.
Location will change to `pkg/storage/stores/shipper/bloomshipper/`